WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListActivity.java

```diff
@@ -20,49 +23,51 @@
 
 import javax.inject.Inject;
 
-public class PostsListActivity extends AppCompatActivity {
+public class PostsListActivity extends AppCompatActivity implements SearchView.OnQueryTextListener {
+    public static final String EXTRA_SEARCH_TERM = "searchTerm";
     public static final String EXTRA_VIEW_PAGES = "viewPages";
     public static final String EXTRA_ERROR_MSG = "errorMessage";
 
     private boolean mIsPage = false;
     private PostsListFragment mPostList;
     private SiteModel mSite;
 
+    private MenuItem mSearchAction;
+    private String mCurrentSearch;
```

Comment:
I am not really sure why we have this `String mCurrentSearch` here, it doesn't look like anything is setting it.

Maybe you were planning to set it inside the `public boolean onQueryTextChange(String query)`?